src/widgets/QDisassemblyView.h

```diff
@@ -82,9 +82,9 @@ class QDisassemblyView final : public QAbstractScrollArea {
 	int removeComment(edb::address_t address);
 	int selectedSize() const;
 	std::shared_ptr<IRegion> region() const;
-	void addComment(edb::address_t address, QString comment);
-	void clearComments();
-	void restoreComments(QVariantList &);
+    void addComment(edb::address_t address, QString comment);
```

Comment:
Nothing seems to have changed here except whitespace?..

Reply:
Exactly, I think it's a mistake made by copying/pasting this line to create the 2 next methods and the different tools I used to write the whole patch (started with nano for Python profile and switched back on QtCreator to enjoy the power of the Qt engine before trying to compile).

Reply:
Resolved with the last commit.